components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -63,7 +63,7 @@ export const PrebuildDetailPage: FC = () => {
     const { toast, dismissToast } = useToast();
     const [currentPrebuild, setCurrentPrebuild] = useState<Prebuild | undefined>();
     const [logNotFound, setLogNotFound] = useState(false);
-    const [selectedTaskId, setSelectedTaskId] = useState<string | undefined>(undefined);
+    const [selectedTaskId, actuallySetSelectedTaskId] = useState<string | undefined>(window.location.hash.slice(1));
```

Comment:
we could improve this with some basic input validation, it is very naive.

Reply:
When opening the the prebuild details page directly (without any taskId passed) we should select the first task, and also focus that task's tab.
Atm I see those rendered, but none has focus, so also not streaming logs. :+1:

Reply:
Of course I forget to test the most important (default) case. Thanks for noticing, just push a fix 🤦